index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Calculator</title>
+	<script src="script.js"></script>
+</head>
+
+<body>
```

Comment:
Tab out your body from the html element. It makes for more readable code. In this case, it isn't entirely necessary, so always use the style format your instructor or employer uses.